### ISSUE TITLE
I'm having issues setting up langflow through docker-compose

### ISSUE THREAD

**Issue by chibexme** [langflow-ai/langflow#2377]
I'm trying to deploy langflow on ubuntu staging server through docker-compose

every other services starts successful except - backend, celeryworker and flower as shown in the below screenshot. 

With error message - /usr/local/bin/python: No module named langflow

![Screenshot 2024-06-26 141841](https://github.com/langflow-ai/langflow/assets/5671057/3f45bb24-6597-462b-a080-9753fba78fee)

when i access the frontend through the browser, i only see the loading image and javascript error as shown on the below screenshot
![Screenshot 2024-06-26 143020](https://github.com/langflow-ai/langflow/assets/5671057/3cdfc999-4dc1-4669-94a1-ab3b70ebaa80)

**Comment by chibexme** [comment]
@dosu, the docker-compose you got your suggestion from was last updated 9months ago, meaning that i doesn't reflect the current way of deploy langflow with docker. 

I've tried your suggestions they didn't work.

**Comment by chibexme** [comment]
@Dosu, for celeryworker and flower services am now getting the below errors:

/app/.venv/lib/python3.12/site-packages/langchain/agents/json_chat/base.py:22: SyntaxWarning: invalid escape sequence '\ '
 """Create an agent that uses JSON to format its logic, build for Chat Models.

Usage: python -m langflow run [OPTIONS]
Try 'python -m langflow run --help' for help.

No such option: --upgrade

**Comment by chibexme** [comment]
Please can someone in the core team update the docker-compose.yml in deploy folder as it was last updated 2months? which definitely doesn't conform to current version 1 deployment requirements.

**Comment by nicoloboschi** [comment]
@chibexme this is going to be fixed in 1.0.6. 
The current issue is "latest" image doesn't point to 1.x release. Plus, there's a problem with the endpoint that has been already fixed in the main branch but not released yet

cc @ogabrielluiz

**Comment by chibexme** [comment]
> @chibexme this is going to be fixed in 1.0.6. The current issue is "latest" image doesn't point to 1.x release. Plus, there's a problem with the endpoint that has been already fixed in the main branch but not released yet
> 
> cc @ogabrielluiz

Alright, will be waiting for the 1.0.6 release.

Thanks.

**Comment by chibexme** [comment]
@ogabrielluiz I can see that version 1.0.6 has been release with many bug fixed, that's impressive.

Regard my concern about docker-compose.yml file in the deploy folder, it still has not been updated as can be seen in the screenshot below
![Screenshot 2024-07-04 160238](https://github.com/langflow-ai/langflow/assets/5671057/221f7c00-cab9-4d3d-bd1c-8f43b58d92ac)


I checked the docker image the frontend is point to, it was updated 9months ago
![Screenshot 2024-07-04 160411](https://github.com/langflow-ai/langflow/assets/5671057/730bef81-bbc1-476d-bde7-80767aceb0d1)

Kindly point me to the right working docker-compose file to deploy langflow from.

Expecting your speedy response.
